include/cantera/thermo/Elements.h

```diff
@@ -10,6 +10,8 @@
 
 #include "cantera/base/ct_defs.h"
 
+using namespace std;
```

Comment:
Yes, exactly. In the header file you need to use the fully qualified name,
then in the cpp file you can have the 'using namespace' and use the
unqualified names.

On Wed, Dec 23, 2015, 3:14 AM Bryan W. Weber <notifications@github.com>
wrote:

> In include/cantera/thermo/Elements.h
> <https://github.com/Cantera/cantera/pull/314#discussion_r48255824>:
>
> > @@ -10,6 +10,8 @@
> >
> >  #include "cantera/base/ct_defs.h"
> >
> > +using namespace std;
>
> This SO answer <http://stackoverflow.com/a/5527669/2449192> says to add
> the qualifier just in the header file. Is that correct then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cantera/cantera/pull/314/files#r48255824>.
>